state-chain/runtime/src/lib.rs

```diff
@@ -907,12 +889,6 @@ type PalletMigrations = (
 	pallet_cf_chain_tracking::migrations::PalletMigration<Runtime, Instance1>,
 	pallet_cf_chain_tracking::migrations::PalletMigration<Runtime, Instance2>,
 	pallet_cf_chain_tracking::migrations::PalletMigration<Runtime, Instance3>,
-	pallet_cf_vaults::migrations::PalletMigration<Runtime, Instance1>,
```

Comment:
let's leave them commented out, the others were commented out because they're easy to forget, just serves as a little reminder